bootstrap-obvious-ci-and-miniconda.py

```diff
@@ -1,103 +0,0 @@
-#!/usr/bin/env python
```

Comment:
All the recent re-renderings point to the commit where this is and it is still available. Have just now rechecked.

https://raw.githubusercontent.com/conda-forge/conda-smithy/e976c7e84bb3c4846e7562afd1a42c4b535b51f5/bootstrap-obvious-ci-and-miniconda.py